packages/generators/app/lib/resources/templates/database-templates/js/database.template

```diff
@@ -0,0 +1,57 @@
+const path = require('path');
```

Comment:
the js and ts templates are not the same ?

Reply:
Not entirely.
The import/export are different.
The path for sql is also not the same. It requires to go up one more level.

Reply:
I'll rephrase. I'm not seeing the same parameters in the DB configs which is the problem I was mentioning :)

Reply:
Got it. Good catch. I forgot to update the `ts` template.